apps/climatemappedafrica/src/lib/data/common/index.js

```diff
@@ -1,6 +1,11 @@
 import { blockify } from "@/climatemappedafrica/lib/data/blockify";
 import { fetchProfile } from "@/climatemappedafrica/lib/hurumap";
 
+// TODO(kilemensi): Use HURUmap APIs (or CMS) to pick geographies we'd like to
+//                  build pages for at build time (It can't be all geographies
+//                  as that will take forever)
+const GEOGRAPHIES = ["af", "ke", "tz"];
```

Comment:
Perharps we can only build the root geography.

Reply:
... yeah, may be:

1. We now have 2 "roots": `/explore` and `/explore/af`. We probably need to add redirect from `/explore` to `/explore/af` (not in this PR)
2. Unless your work with Muma reduces the amount of data we need to fetch considerably, we'd still need to build some "key" countries/regions since HURUmap data fetching is still (very) slow.